test/loader/serializeInstructionTest.cpp

```diff
@@ -385,11 +385,24 @@ TEST(SerializeInstructionTest, SerializeCallControlInstruction) {
   //   4.  Serialize Call_ref instruction with valid type index.
   //   5.  Serialize Return_call_ref instruction with valid type and table.
   //   6.  Serialize Return_call_ref without Func-Ref proposal.
+  //   7.  Serialize Throw_ref instruction.
```

Comment:
The EH instructions are not call control instructions.
I mean move the EH instruction test set to the "NEXT" of `SerializeCallControlInstruction`, not combine them into the test.